lib/date_range_form_field.dart

```diff
@@ -59,11 +59,6 @@ class DateRangeField extends FormField<DateTimeRange> {
               final InputDecoration inputDecoration = decoration
                   .copyWith(enabled: enabled)
                   .applyDefaults(Theme.of(state.context).inputDecorationTheme);
-              // if (state.value == null) {
```

Comment:
Normally it would be better to put this into a separate commit and separate PR as it is not conceptually related to purpose of this PR. Sometimes people refer to this kind of a change as a 'non-functional change'.